wafw00f/plugins/anquanbao.py

```diff
@@ -9,7 +9,10 @@
 
 def is_waf(self):
     schemes = [
+
+        # just added another extra signature used by Anquanbao
         self.matchHeader(('X-Powered-By-Anquanbao', '.+?')),
+        self.matchHeader(('X-Powered-By-Anquanbao', 'MISS')),
```

Comment:
this seems redundant since the previous match would take precedence. Was the previous line not working for you?